includes/Vendor/SetupWizard.php

```diff
@@ -319,7 +321,7 @@ public function dokan_setup_store() {
                         <input type="text" id="calc_shipping_state" name="address[state]" value="<?php echo esc_attr( $address_state ); ?>" / placeholder="<?php esc_attr_e( 'State Name', 'dokan-lite' ); ?>">
                         <span class="error-container">
                             <?php
-                            if ( ! empty( $_POST['error_address[state]'] ) ) {
+                            if ( ! empty( $_POST['error_address[state]'] ) ) { // phpcs:ignore
```

Comment:
Assign the `$_POST` to a variable  like `$request_data` to use safely instead of ignoring *PHPCS* everywhere.